tests.py

```diff
@@ -92,19 +92,23 @@ def tearDownClass(cls):
     shutil.rmtree(os.path.join('test_data', 'docker'))
 
   @classmethod
-  def _setup(cls, driver, driver_class):
+  def _setup(cls, driver, driver_class, storage_version=2):
     """Internal method to set up the TestCase on a specific storage."""
     cls.driver = driver
     docker_directory_path = os.path.join('test_data', 'docker')
     if not os.path.isdir(docker_directory_path):
-      docker_tar = os.path.join('test_data', cls.driver+'.tgz')
+      docker_tar = os.path.join(
+          'test_data', '{0:s}.v{1:d}.tgz'.format(driver, storage_version))
       tar = tarfile.open(docker_tar, 'r:gz')
       tar.extractall('test_data')
       tar.close()
     cls.de_object = de.DockerExplorer()
+    cls.de_object.docker_version = storage_version
```

Comment:
I removed this line.

I agree this should read "storage_version" instead of docker_version, but this is going to be quite a large PR, tracking in #24